tests/coalaTest.py

```diff
@@ -65,6 +65,16 @@ def test_did_nothing(self):
         self.assertIn('Nothing to do.', stderr)
         self.assertFalse(stdout)
 
+    def test_format(self):
+        retval, stdout, stderr = execute_coala(coala.main, 'coala', '--ci',
+                                               '--format', '-b',
+                                               'SpaceConsistencyTestBear',
+                                               '-f', '*.py'
+                                               )
+        self.assertEqual(retval[0], 0)
+        self.assertEqual(retval[1], 0)
+        self.assertIn('Executing section cli...', stdout)
```

Comment:
Please assert stderr also as empty. we separated stdout and stderr return for the reason that both can be used individually.